.github/workflows/ci.yml

```diff
@@ -94,6 +94,11 @@ jobs:
     - name: Run tests
       run: |
         ./y.sh test --release --clean --build-sysroot ${{ matrix.commands }}
+      
+    - name: Run y.sh cargo build
+      run: |
+        export RUSTFLAGS="-Zcodegen-backend=$(pwd)/target/release/librustc_codegen_gcc.so"
```

Comment:
Instead of doing this, I would prefer if you moved this task below `Build`. This would require to split the `Build` task so that the cleaning part happens in a new task after `Run y.sh cargo build`.